x-pack/plugins/uptime/public/components/overview/monitor_list/columns/monitor_status_column.test.tsx

```diff
@@ -227,33 +225,53 @@ describe('MonitorListStatusColumn', () => {
     ];
   });
 
-  it('provides expected tooltip and display times', () => {
-    const component = shallowWithIntl(
-      <MonitorListStatusColumn status="up" timestamp="2314123" summaryPings={[]} />
+  it('provides expected tooltip and display times', async () => {
+    const { getByText } = render(
+      <EuiThemeProvider darkMode={false}>
+        <MonitorListStatusColumn status="up" timestamp="2314123" summaryPings={[]} />
+      </EuiThemeProvider>
     );
-    expect(component).toMatchSnapshot();
+
+    const timestamp = getByText('Sept 4, 2020', { exact: false });
+    expect(timestamp.innerHTML).toEqual('Checked Sept 4, 2020  9:31:38 AM');
+
+    fireEvent.mouseOver(timestamp);
+
+    await waitFor(() => screen.getByText('Thu May 09 2019 10:15:11 GMT-0400'));
   });
 
   it('can handle a non-numeric timestamp value', () => {
-    const component = shallowWithIntl(
-      <MonitorListStatusColumn status="up" timestamp={new Date().toString()} summaryPings={[]} />
+    const { getByText } = render(
+      <EuiThemeProvider darkMode={false}>
+        <MonitorListStatusColumn status="up" timestamp={new Date().toString()} summaryPings={[]} />
+      </EuiThemeProvider>
+    );
+
+    expect(getByText('Sept 4, 2020', { exact: false }).innerHTML).toEqual(
+      'Checked Sept 4, 2020  9:31:38 AM'
     );
-    expect(component).toMatchSnapshot();
   });
 
-  it('will display location status', () => {
-    const component = shallowWithIntl(
-      <MonitorListStatusColumn
-        status="up"
-        timestamp={new Date().toString()}
-        summaryPings={summaryPings}
-      />
+  it('will display location status', async () => {
+    const { getByText } = render(
+      <EuiThemeProvider darkMode={false}>
+        <MonitorListStatusColumn
+          status="up"
+          timestamp={new Date().toString()}
+          summaryPings={summaryPings}
+        />
+      </EuiThemeProvider>
     );
-    expect(component).toMatchSnapshot();
+
+    const locationsContainer = getByText('in 1/3 locations', { exact: false });
+
+    fireEvent.mouseOver(locationsContainer);
+
+    await waitFor(() => screen.getByText('Up in BerlinDown in Islamabad, st-paul'));
```

Comment:
Should there be a space between?

Reply:
Good question, I'll have a look in a real-world case. It could just be the way the raw html looks.

Reply:
It was indeed a bug; we could've tackled it in a separate issue but I fixed it in https://github.com/elastic/kibana/pull/73709/commits/2a22f2981543bd44620f8a6a537aa5b8ea0ed1ae.